El título en castellano

aaguadoh opened the pull request.

aaguadoh committed:
El título en castellano

aaguadoh committed:
Merge branch 'master' into master

borja commented:
Alfredo (@aaguadoh )

Hay dos entradas de tipo `h1` y queda redundante.
Por favor edita una vez más, para que no se repita.

¡Saludos!

aaguadoh committed:
Eliminada etiqueta duplicada h1

Modificado el fichero en el fork. No he podido descargar todos los ficheros (por si hubieran cambiado) ya que el upstream lo tengo montado en mi portatil y desde la web no tengo claro cómo hacerlo

aaguadoh committed:
Solucionado

Eliminada etiqueta duplicada h1

borja commented:
Buena idea, pero quedan conflictos sin corregir.

¡Saludos!

aaguadoh committed:
Merge branch 'master' into master

aaguadoh commented:
Se ha solucionado el conflicto

aaguadoh committed:
Merge branch 'master' into master

aaguadoh commented:
Acabo de volver a arreglar el conflicto existente dejando sólo un h1. Mantengo el 1º del compañero y elimino el segundo

aaguadoh committed:
Merge branch 'master' into master

aaguadoh commented:
He vuelto a resolver el conflicto. También he eliinado una serie de líneas duplicadas

borja commented:
Excelente trabajo @aaguadoh
Has aprobado la práctica.

Puedes pasarte para revisar el examen
:smile:
